x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -45,10 +45,14 @@ export class HeadlessChromiumDriver {
   private readonly inspect: boolean;
   private readonly networkPolicy: NetworkPolicy;
 
+  private listenersAttached: boolean;
```

Comment:
This can be initialized here with `private listenersAttached = false;` and then it doesn't need to be initialized in the constructor